### PR TITLE
IdentityContext tweaks

### PR DESCRIPTION
**Pull request by sdirosa**
Some minor refinement to IdentityContext. 

The name attribute has been renamed to value, for consistency with the other contexts. Thus the `id` attribute now determines the scope and `value` is where developers will actually set their identity hash.

Factories and classes have been regenerated accordingly. Tests have been updated to reflect this change as well.

### CODE REVIEW

File: schema/base_schema.json5 [hunk 1]
```diff
@@ -261,8 +261,8 @@
       "description": "A Global Context to track the identity of users across sessions, platforms, devices. Multiple can be present.",
```
Comment (jansentom) [inline schema/base_schema.json5:261]:
```suggestion
      "description": "A Global Context to track the identity of users across sessions, platforms, devices. Multiple can be present. The `id` field is used to specify the type of identification e.g. backend, md5(email), supplier_cookie. The `value` field should contain the unique identifier within that scope.",
```

Comment (sdirosa) [reply schema/base_schema.json5]: Applied and regenerated Collector's schema and SDK factories

File: backend/objectiv_backend/schema/schema.py [hunk 2]
```diff
@@ -295,30 +295,33 @@ def __init__(self,
 class IdentityContext(AbstractGlobalContext):
     """
         A Global Context to track the identity of users across sessions, platforms, devices. Multiple can be present.
+    The `id` field is used to specify the type of identification e.g. backend, md5(email), supplier_cookie.
+    The `value` field should contain the unique identifier within that scope.
```
Comment (sdirosa) [inline backend/objectiv_backend/schema/schema.py:299]: Indentation is meh, but this is a generator.js issue

File: backend/objectiv_backend/schema/schema.py [hunk 3]
```diff
@@ -295,30 +295,33 @@ def __init__(self,
...
         """
-        AbstractGlobalContext.__init__(self, name=name, id=id, **kwargs)
+        AbstractGlobalContext.__init__(self, value=value, id=id, **kwargs)
 
 
 class AbstractLocationContext(AbstractContext, ABC):
     """
-        AbstractLocationContext are the abstract parents of all Location Contexts. Location Contexts are meant to describe where an event originated from in the visual UI.
+        AbstractLocationContext are the abstract parents of all Location Contexts.
+    Location Contexts are meant to describe where an event originated from in the visual UI.
```
Comment (sdirosa) [inline backend/objectiv_backend/schema/schema.py:324]: Unrelated fix, this description was too long for schema.py

File: backend/objectiv_backend/schema/schema.py [hunk 4]
```diff
@@ -503,7 +506,8 @@ def __init__(self, id: str, **kwargs: Optional[Any]):
 
 class ContentContext(AbstractLocationContext):
     """
-        A Location Context that describes a logical section of the UI that contains other Location Contexts. Enabling Data Science to analyze this section specifically.
+        A Location Context that describes a logical section of the UI that contains other Location Contexts.
+    Enabling Data Science to analyze this section specifically.
```
Comment (sdirosa) [inline backend/objectiv_backend/schema/schema.py:510]: Same as above
